libs/community/langchain_community/embeddings/fastembed.py

```diff
@@ -48,12 +52,24 @@ class FastEmbedEmbeddings(BaseModel, Embeddings):
     The available options are: "default" and "passage"
     """
 
+    batch_size: int = 256
```

Comment:
(nit) If FastEmbed default changes, this will be out of date. Would prefer to have this Optional and not pass in in the None case.

Reply:
Will do.

Reply:
I see this could also be applicable to `model_name`, `max_length`.
Leading to to quite some duplicated code due to the conditionals.

So I propose to keep it. WDYT?